### DISCUSSION TITLE
[WinError 216] This version of %1 is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher Validate unsuccessful login

### DISCUSSION THREAD

**Discussion by snalwoga** [community/community#37912] [Other Feature Feedback, Questions, & Ideas]
I am working on a project in pycharm but I keep getting the following errors as seen below.
Can someone help me?

 Thanks 

[info] Opening browser 'firefox' to base url 'with morgage payment url'.
[info (+0.04s)] Firefox driver log is always forced to to: C:\Users\smnal\PycharmProjects\pythonProject4\robotframework\log\geckodriver-19.log
[info (+0.43s)] Cannot capture screenshot because no browser is open.
[FAIL] OSError: [WinError 216] This version of %1 is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher
Validate unsuccessful login

**Comment by ossanna16** [comment]
Hi @snalwoga,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on PyCharm. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
